samples/automl/automlVisionModel.js

```diff
@@ -0,0 +1,607 @@
+        );
+        console.log(
+          `\tStop reason: `,
+          model.imageClassificationModelMetadata.stopReason
+        );
+      }
+      console.log(`Model create time:`);
+      console.log(`\tseconds: `, model.createTime.seconds);
+      console.log(`\tnanos: `, model.createTime.nanos);
+      console.log(`Model update time:`);
+      console.log(`\tseconds: `, model.updateTime.seconds);
+      console.log(`\tnanos: `, model.updateTime.nanos);
+      console.log(`Model deployment state: `, model.deploymentState);
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_getModel]
+}
+
+function listModelEvaluations(projectId, computeRegion, modelId, filter_) {
+  // [START automl_vision_listModelEvaluations]
+  const automl = require(`@google-cloud/automl`);
+  const util = require(`util`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const modelId = `id of the model, e.g. “ICN12345”`;
+  // const filter_ = `filter expressions, must specify field, e.g. “imageClassificationModelMetadata:*”`;
+
+  // Get the full path of the model.
+  const modelFullId = client.modelPath(projectId, computeRegion, modelId);
+
+  // List all the model evaluations in the model by applying filter.
+  client
+    .listModelEvaluations({parent: modelFullId, filter: filter_})
+    .then(responses => {
+      const element = responses[0];
+      for (let i = 0; i < element.length; i += 1) {
+        console.log(util.inspect(element[i], false, null));
+      }
+    })
+    .catch(err => {
+      console.error(err);
+    });
+  // [END automl_vision_listModelEvaluations]
+}
+
+function getModelEvaluation(
+  projectId,
+  computeRegion,
+  modelId,
+  modelEvaluationId
+) {
+  // [START automl_vision_getModelEvaluation]
+  const automl = require(`@google-cloud/automl`);
+  const util = require(`util`);
+
+  const client = new automl.v1beta1.AutoMlClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const projectId = `The GCLOUD_PROJECT string, e.g. "my-gcloud-project"`;
+  // const computeRegion = `region-name, e.g. "us-central1"`;
+  // const modelId = `id of the model, e.g. “ICN12345”`;
+  // const modelEvaluationId = `Id of your model evaluation, e.g “ICN12345”
+
+  // Get the full path of the model evaluation.
+  const modelEvaluationFullId = client.modelEvaluationPath(
+    projectId,
+    computeRegion,
+    modelId,
+    modelEvaluationId
+  );
+
+  // Get complete detail of the model evaluation.
+  client
+    .getModelEvaluation({name: modelEvaluationFullId})
+    .then(responses => {
+      const response = responses[0];
+      console.log(util.inspect(response, false, null));

```

Comment:
Can we display a subset of `response` properties here, instead of using `util.inspect`?